Makefile

```diff
@@ -19,12 +20,14 @@ install-docker: install-core
 	install -D -m 644 ${DOCKER}-storage-setup.service ${UNITDIR}/${DOCKER}-storage-setup.service
 	if [ ! -f ${SYSCONFDIR}/${DOCKER}-storage-setup ]; then \
 		install -D -m 644 ${DOCKER}-storage-setup-override.conf ${SYSCONFDIR}/${DOCKER}-storage-setup; \
+		echo "CONTAINER_THINPOOL=docker-pool" >> ${SYSCONFDIR}/${DOCKER}-storage-setup; \
 	fi
 
 .PHONY: install-core
 install-core:
-	install -D -m 755 ${DOCKER}-storage-setup.sh ${BINDIR}/${DOCKER}-storage-setup
-	install -D -m 644 ${DOCKER}-storage-setup.conf ${DSSLIBDIR}/${DOCKER}-storage-setup
-	install -D -m 755 libdss.sh ${DSSLIBDIR}/libdss.sh
-	install -D -m 755 dss-child-read-write.sh ${DSSLIBDIR}/dss-child-read-write
-	install -D -m 644 ${DOCKER}-storage-setup.1 ${MANDIR}/man1/${DOCKER}-storage-setup.1
+	install -D -m 755 ${CONTAINER}-storage-setup.sh ${BINDIR}/${CONTAINER}-storage-setup
+	(cd ${BINDIR}; ln -sf ${CONTAINER}-storage-setup ${DOCKER}-storage-setup)
```

Comment:
That sounds like a real corner case where docker package is old while container-storage-setup is newer.